dbbackup/utils.py

```diff
@@ -83,10 +116,12 @@ def encrypt_file(inputfile):
             always_trust = settings.GPG_ALWAYS_TRUST
             g = gnupg.GPG()
             result = g.encrypt_file(inputfile, output=filepath,
-                recipients=settings.GPG_RECIPIENT, always_trust=always_trust)
+                                    recipients=settings.GPG_RECIPIENT,
+                                    always_trust=always_trust)
             inputfile.close()
             if not result:
-                raise Exception('Encryption failed; status: %s' % result.status)
+                msg = 'Encryption failed; status: %s' % result.status
+                raise Exception(msg)
```

Comment:
It's because I want to respect pep8 and 80 characters per line.

> Le 27 juil. 2015 à 00:34, benjaoming <notifications@github.com> a écrit :
> 
> In dbbackup/utils.py:
> 
> >              inputfile.close()
> >              if not result:
> > -                raise Exception('Encryption failed; status: %s' % result.status)
> > +                msg = 'Encryption failed; status: %s' % result.status
> > +                raise Exception(msg)
> Not that I want to block this or anything, but it's a bit odd to assign a variable for something used once in a function call :)
> 
> —
> Reply to this email directly or view it on GitHub.
>